notebook/static/notebook/js/notebook.js

```diff
@@ -1176,7 +1176,17 @@ define(function (require) {
         var len = this.ncells();
         return this.insert_cell_below(type,len-1);
     };
-
+    
+    
```

Comment:
Can you add a documentation string before the function to describe what it does? You can look at the documentation string above `to_code` for guidance.